src/Feliz.Plotly/Feliz.Plotly.fsproj

```diff
@@ -22,14 +34,42 @@
     <Compile Include="Plotly.fs" />
     <Compile Include="PlotlyFunctions.fs" />
     <None Include="paket.references" />
-    <None Include="paket.template" />
-    <None Include="../../readme.md" Pack="true" PackagePath="/"/>
+    <None
+        Include="../../readme.md"
+        Pack="true"
+        PackagePath="/" />
+  </ItemGroup>
+
+  <ItemGroup Label="Fable compilation items">
+    <!-- Fable needs the sources, as well as any generated JS files, to end up in the `fable`
+    directory of the package. The paket.template in this repo specifies
+      * source files
+      * project file
+      * JS files in the Locales directory => fable/Locales -->
+    <Content
+        Include="@(Compile)"
+        Pack="true"
+        PackagePath="fable/%(RelativeDir)%(Filename)%(Extension)" />
+    <Content
+        Include="$(MSBuildThisFileFullPath)"
+        Pack="true"
+        PackagePath="fable/" />
+    <Content
+        Include="Locales/*.js"
+        Pack="true"
+        PackagePath="fable/%(RelativeDir)%(Filename)%(Extension)" />
```

Comment:
We can declare the package paths for all of the files that we want in one place and let the build system take care of actually copying them.